tests/ImageSharp.Tests/Formats/Jpg/DCTTests.cs

```diff
@@ -169,7 +288,10 @@ public void TransformFDCT(int seed)
                 var temp1 = new float[64];
                 var temp2 = default(Block8x8F);
 
+                // reference
                 ReferenceImplementations.LLM_FloatingPoint_DCT.FDCT2D_llm(src, expectedDest, temp1, downscaleBy8: true);
+
+                // testee
```

Comment:
There are 3 possible paths in `TransformFDCT` depending on hardware capabilities (Vector4 vs AVX vs AVX+FMA).

To stress the method in a robust manner (including the  selection logic), we need to do out of process test execution with capabilities being selectively disabled using environment variables. We have a tool `FeatureTestRunner` to do this:

https://github.com/SixLabors/ImageSharp/blob/97dde7fb2f0ac97b5b1f9e6b0555a53d2e270c46/tests/ImageSharp.Tests/Common/SimdUtilsTests.Shuffle.cs

Note that it swallows errors and produces false greens in Visual Studio (#1376), but should work on CI.

Reply:
Same to IDCT then, will try to wrap my head around `FeatureTestRunner` today.

Reply:
And while we are here, there's two tests which test IDCT transform with `LLM_FloatingPoint_DCT.TransformIDCT` and another one with `AccurateDCT.TransformIDCT`. Pre-pr tests used for LLM implementation for testing IDCT parts, I used same reference for new methods. So there are some questions:

1. Should I use LLM of Accurate for tests? Both?
2. If only one reference is used, should the other one exists in test code at all?
3. If both used, should both asserts be checks at the same test method or 2 different methods per 1 test?